core/tests/protractor_desktop/userJourneys.js

```diff
@@ -33,8 +33,6 @@ var users = require('../protractor_utils/users.js');
 var waitFor = require('../protractor_utils/waitFor.js');
```

Comment:
Can we move the "Site Language" test to editorAndPlayer or editorFeatures? We can then make entire userJourneys accessible from both desktop and mobile!

Here is what I understood from "Site Language"
1)Before all: Create exp and add exp to collection
2)Before each: Go to /about and select English as language
3)Each test has a variation of selecting different language and expecting different elements on the page
to be in that selected language.
This sounds like an editorAndPlayer to me since the test 
1) change language 
2) navigate to collection player or exploration player 
3) check if elements on page's displayed text match the expected language.

Let me know if I interpreted things correctly @apb7  :)

Reply:
p.s we can also test for language changes on mobile using a pre-loaded exp too. That can be a TO-DO imo.

Reply:
Hi Viet, userJourneys will still not be accessible from both mobile and desktop. I would like to bring in a few points here:
1. I have seen the AdminPage break down sometimes for mobile. Also, as I had explained earlier, updating role from mobile via the AdminPage can flake out sometimes due to popups blocking the "Update Role" button. Therefore, we used an alternative function, as you had suggested, to sign a user as an admin.
2. In the static pages tour suite, the "About" dropdown isn't visible for mobile. So that spec goes out.
This is just a general overview. I haven't looked into the preference page object. We are aiming to let in the core learner functionality at the moment so I feel these specs do justice to that but we can certainly look into incorporating your suggestions :)

Reply:
I think we are covering both these specs in our tests already -- there is a file `navigation.js` which uses the **sidenav** to navigate to all the links, so the static pages tour is already being covered. Regarding the `it ('should create admins on mobile')`, if the idea is to use createAdminOnMobile(), then we are already using it in `beforeEach` and `beforeAll` in some of the tests but not explicitly.

Reply:
I mean if parts of userJourneys is already covered by other similar tests on protractor_mobile then no need for repeating it! I just wanted PreferencesPage test to be ran on mobile. Thats all :) I am good for now.

Reply:
Sounds good then, thanks :)